### PR TITLE
Add a tooltip for the disabled "Start call" button

### LINKED ISSUE #3436: Grey out "Start call" button if user is not allowed to start a call (and provide explanatory tooltip)
If a user is not allowed to start a call we should hide the "start call" button completely, not just disable it. I don't think it makes sense to show a disabled button for a action a user will never be able to perform.

cc @nextcloud/designers

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #3436

### CODE REVIEW

File: src/components/TopBar/CallButton.vue [hunk 1]
```diff
@@ -132,6 +132,18 @@ export default {
 			return t('spreed', 'Start call')
 		},
 
+		startCallToolTip() {
+			if (this.callButtonTooltipText) {
+				return this.callButtonTooltipText
+			}
+
+			if (!this.conversation.canStartCall && !this.conversation.hasCall) {
+				return t('spreed', 'You can only join the call after a moderator started it.')
```
Comment (danxuliu) [inline src/components/TopBar/CallButton.vue:141]: Maybe it should be just _You can not start a call in this conversation_ or something like that, without any mention to moderators, as starting the calls could be limited only for guests but allowed for moderators and regular users too?

Comment (nickvergessen) [reply src/components/TopBar/CallButton.vue:141]: Well for the guest this doesn't really matter

Comment (nickvergessen) [reply src/components/TopBar/CallButton.vue:141]: And I guess they can figure they can't because the button is disabled.

**Pull request merged**